### DISCUSSION TITLE
Target Disk Mode

### DISCUSSION THREAD

**Discussion by kstella** [iterate-ch/cyberduck#16321] [Q&A]
I'm having trouble copying a large directory via macOS (Ventura) and _target disk mode_. (It gives up with a resume dialog that never connects again)
A colleague suggested I use Cyberduck. I've only used it for FTP so I'd like some advice how I can utilize it copy a large directory.

**Comment by tatze96** [comment]
Please add a screenshot of the error message. Ensure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.
